### PR TITLE
Add assets tab in devices view

### PR DESCRIPTION
**Pull request by bojanutkovic**
Signed-off-by: Bojan Utkovic <bojan.utkovic@comtrade.com>

### CODE REVIEW

File: console/src/main/java/org/eclipse/kapua/app/console/client/device/AssetDetailsTable.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
+package org.eclipse.kapua.app.console.client.device;
```
Comment (ctron) [inline console/src/main/java/org/eclipse/kapua/app/console/client/device/AssetDetailsTable.java]: There is a copyright header missing. Same for the other files.

File: console/src/main/java/org/eclipse/kapua/app/console/client/device/AssetDetailsTable.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+import com.extjs.gxt.ui.client.widget.toolbar.ToolBar;
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.user.client.Element;
+import com.google.gwt.user.client.rpc.AsyncCallback;
+
+public class AssetDetailsTable extends LayoutContainer{
+   
+    private GwtSession currentSession;
+    private ContentPanel tableContainer;
+    private ToolBar m_toolBar;
```
Comment (ctron) [inline console/src/main/java/org/eclipse/kapua/app/console/client/device/AssetDetailsTable.java]: The "m_" style is no longer used. Same for the other locations.

File: console/src/main/java/org/eclipse/kapua/app/console/client/device/AssetDetailsTable.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    @Override
+    protected void onRender(Element parent, int index) {
+        super.onRender(parent, index);
+        setLayout(new FitLayout());
+        setBorders(false);
+        initDetailsTable();
+        add(tableContainer);
+    }
+    public void onUnload() {
+        super.onUnload();
```
Comment (ctron) [inline console/src/main/java/org/eclipse/kapua/app/console/client/device/AssetDetailsTable.java]: Seems to be unnecessary?!

File: console/src/main/java/org/eclipse/kapua/app/console/client/device/DeviceValuesTab.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    
+    public void setDevice(GwtDevice selectedDevice){
+        m_dirty = true;
+        m_selectedDevice = selectedDevice;
+    }
+    
+    public void refresh(){
+        if (m_dirty && m_initialized) {
+            m_dirty = false;
+            if (m_selectedDevice == null) {
```
Comment (ctron) [inline console/src/main/java/org/eclipse/kapua/app/console/client/device/DeviceValuesTab.java]: Is there something missing?

**Comment by lorthirk** [comment]
Looks like all the change requests have been addressed. @ctron?

**Pull request merged**